openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -1,3 +1,14 @@
+export function initAddBookImport () {
+    $('.list-books a').click(function() {
+        var li = $(this).parents('li:first');
+        $('input#work').val(`/works/${li.attr('id')}`);
+        $('form#addbook').submit();
+    });
+    $('#bookAddCont').click(function() {
```

Comment:
FYI (for future patches): https://github.com/internetarchive/openlibrary/issues/5198

Reply:
I think you accidentally added that function. I removed it

Reply:
Thanks!